src/django/api/templates/mail/report_result_body.html

```diff
@@ -10,22 +10,32 @@
             Hi there,
         </p>
         <p>
-            Thanks for reporting {{ facility_name }} facility as {{ closure_state|lower }}.
-            {% if is_closure and is_rejected %}We have rejected this report and the facility's profile in OS Hub has not been changed.{% endif %}{% if is_closure and is_confirmed %}The facility profile in the Open Supply Hub has now been updated and the facility is marked as closed.{% endif %}{% if is_reopening and is_rejected %}We have rejected this report and the facility's profile in OS Hub has not been changed.{% endif %}{% if is_reopening and is_confirmed %}The facility profile in the Open Supply Hub has now been updated and the facility is no longer marked as closed.{% endif %}
+            Thanks for reporting {{ facility_name }} facility as {{
+            closure_state|lower }}. {% if is_closure and is_rejected %}We have
```

Comment:
It looked a bit unusual to split the template tag across lines.

I tested rejecting a facility closure report ran into an exception. 

>django_1     | django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 20: 'endif'. Did you forget to register or load this tag?

I reverted to the line that was here before, remove the "the" and it resolved the exception.

```
            {% if is_closure and is_rejected %}We have rejected this report and the facility's profile in OS Hub has not been changed.{% endif %}{% if is_closure and is_confirmed %}The facility profile in the Open Supply Hub has now been updated and the facility is marked as closed.{% endif %}{% if is_reopening and is_rejected %}We have rejected this report and the facility's profile in OS Hub has not been changed.{% endif %}{% if is_reopening and is_confirmed %}The facility profile in Open Supply Hub has now been updated and the facility is no longer marked as closed.{% endif %}
```